### PR TITLE
Fixing the incorrect sizing error

### PR DESCRIPTION
**Pull request by Dzejkop**

### CODE REVIEW

File: Assets/PatchKit Patcher/Editor/CustomBuilding.cs [hunk 1]
```diff
@@ -78,6 +78,25 @@ private static string PatcherExecutableName(BuildTarget target)
             }
         }
 
+        private static void EnforceCorrectScreenSizing()
+        {
+            if (PlayerSettings.defaultScreenWidth != Constants.ScreenWidth
```
Comment (witcher112) [inline Assets/PatchKit Patcher/Editor/CustomBuilding.cs]: I think this is a breaking change as we recommend to change player settings to adjust the patcher resolution.

File: Assets/PatchKit Patcher/Scripts/UI/BorderlessWindow.cs [hunk 2]
```diff
@@ -107,9 +107,18 @@ private Vector2 GetScreenResoultion()
         private Vector2 _dragWindowStartPosition;
 #endif
 
+        private void EnsureCorrectScreenSizing()
+        {
+#if !UNITY_2017_4_OR_NEWER
+            Screen.SetResolution(Constants.ScreenWidth, Constants.ScreenHeight, false);
```
Comment (witcher112) [inline Assets/PatchKit Patcher/Scripts/UI/BorderlessWindow.cs]: Can we somehow get values from player settings there?

File: Assets/PatchKit Patcher/Scripts/UI/BorderlessWindow.cs [hunk 3]
```diff
@@ -107,9 +107,18 @@ private Vector2 GetScreenResoultion()
         private Vector2 _dragWindowStartPosition;
 #endif
 
+        private void EnsureCorrectScreenSizing()
+        {
+#if !UNITY_2017_4_OR_NEWER
```
Comment (witcher112) [inline Assets/PatchKit Patcher/Scripts/UI/BorderlessWindow.cs]: What about 2017.4 and later?

File: Assets/PatchKit Patcher/Scripts/UI/BorderlessWindow.cs (hunk 2 above)
Comment (Dzejkop) [reply Assets/PatchKit Patcher/Scripts/UI/BorderlessWindow.cs]: Yeah

File: Assets/PatchKit Patcher/Scripts/UI/BorderlessWindow.cs (hunk 3 above)
Comment (Dzejkop) [reply Assets/PatchKit Patcher/Scripts/UI/BorderlessWindow.cs]: Well, according to our information the issue doesn't appear in versions later or equal to 2017.4.

But then again, maybe we should correct for it anyway.

File: Assets/PatchKit Patcher/Scripts/UI/BorderlessWindow.cs (hunk 2 above)
Comment (Dzejkop) [reply Assets/PatchKit Patcher/Scripts/UI/BorderlessWindow.cs]: Oh, actually I think we can't as the UnityEditor isn't available at runtime.

File: Assets/PatchKit Patcher/Scripts/UI/BorderlessWindow.cs (hunk 3 above)
Comment (witcher112) [reply Assets/PatchKit Patcher/Scripts/UI/BorderlessWindow.cs]: I think we should.

**Pull request merged**